+p_xml: 'version="1.0" encoding="UTF-8" standalone="yes" '
+directive: DOCTYPE bugzilla SYSTEM "http://bugzilla.fieldtriptoolbox.org/page.cgi?id=bugzilla.dtd"
bugzilla:
  +@version: 4.4.1
  +@urlbase: http://bugzilla.fieldtriptoolbox.org/
  +@maintainer: r.oostenveld@donders.ru.nl
  bug:
    bug_id: "1936"
    creation_ts: 2013-01-17 15:05:00 +0100
    short_desc: warning .img file couldn't be opened when ft_volumesegment is used
    delta_ts: 2013-02-23 20:57:14 +0100
    reporter_accessible: "1"
    cclist_accessible: "1"
    classification_id: "1"
    classification: Unclassified
    product: FieldTrip
    component: core
    version: unspecified
    rep_platform: PC
    op_sys: Linux
    bug_status: CLOSED
    resolution: FIXED
    bug_file_loc:
    status_whiteboard:
    keywords:
    priority: P3
    bug_severity: normal
    target_milestone: '---'
    everconfirmed: "1"
    reporter:
      +content: lilla.magyari
      +@name: Lilla Magyari
    assigned_to:
      +content: j.schoffelen
      +@name: Jan-Mathijs Schoffelen
    cc: j.schoffelen
    comment_sort_order: oldest_to_newest
    long_desc:
      - +@isprivate: "0"
        commentid: "9034"
        comment_count: "0"
        who:
          +content: lilla.magyari
          +@name: Lilla Magyari
        bug_when: 2013-01-17 15:05:17 +0100
        thetext: "when I run /test/test_ft_volumesegment.m\n\nI get this warning on the screen:\n\nWarning: could not open /tmp/tpe0dcbe0a_8044_4c4e_b0b9_39b41b6c066a.img \n> In fileio/private/warning_once at 116\n  In fileio/private/filetype_check_header at 54\n  In ft_filetype at 328\n  In ft_write_mri at 54\n  In ft_volumesegment at 293\n  In test_ft_volumesegment at 26 \n\nLilla"
      - +@isprivate: "0"
        commentid: "9262"
        comment_count: "1"
        who:
          +content: j.schoffelen
          +@name: Jan-Mathijs Schoffelen
        bug_when: 2013-01-29 12:20:31 +0100
        thetext: "cause: ft_filetype calls filetype_check_header which fails to find the specified file, because it does not exist yet (the call to ft_filetype occurs in ft_write_volume (called from align_ctf2spm), because SPM needs to temporarily write the volumes). \nExplicitly providing the dataformat in the call to ft_write_volume gets rid of the warning."
      - +@isprivate: "0"
        commentid: "9264"
        comment_count: "2"
        who:
          +content: j.schoffelen
          +@name: Jan-Mathijs Schoffelen
        bug_when: 2013-01-29 12:22:30 +0100
        thetext: added dataformat as input to ft_write_mri in revision 7420, for align_ctf2spm and align_itab2spm
      - +@isprivate: "0"
        commentid: "9440"
        comment_count: "3"
        who:
          +content: j.schoffelen
          +@name: Jan-Mathijs Schoffelen
        bug_when: 2013-02-15 16:30:09 +0100
        thetext: "bash-3.2$ svn commit ft_volumerealign.m ft_volumesegment.m utilities/private/align_ctf2spm.m utilities/private/align_itab2spm.m \nSending        ft_volumerealign.m\nSending        ft_volumesegment.m\nSending        utilities/private/align_ctf2spm.m\nSending        utilities/private/align_itab2spm.m\nTransmitting file data ....\nCommitted revision 7479.\n\n\nThese files needed to be changed, too."
